app/src/main/java/com/cornellappdev/android/eatery/MapsActivity.java

```diff
@@ -199,6 +210,15 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
             }
         }
     }
+    private Bitmap bitmapDescriptorFromVector(Context context, int vectorResId) {
```

Comment:
Add comment about what this method does or add link to where you got this method from